sh/shadowenv.bash.in

```diff
@@ -1,5 +1,5 @@
 __shadowenv_hook() {
-  eval "$("@SELF@" hook "${__shadowenv_data}")"
+  "@SELF@" hook "${__shadowenv_data}" | source /dev/stdin
```

Comment:
I had to change this in order for the parent PID to be the shell session, and not a subshell.